public/app/features/integrated-alerting/components/AlertRuleTemplate/AddAlertRuleTemplateModal/AddAlertRuleTemplateModal.styles.ts

```diff
@@ -0,0 +1,16 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = ({ spacing }: GrafanaTheme) => ({
+  formWrapper: css`
+    #input-yaml-id {
```

Comment:
Do we want to have the ids in our code?

Reply:
It's used to change the default height of the textarea. Do you think it would be better to select by element type @atymchuk ? Or by data-qa?

Reply:
I guess I can use classname here instead of doing it with the ID, thanks for pointing @atymchuk, will update it.

Reply:
I appreciate it! This will allow us to be consistent with the existing codebase, which doesn't seem to favor html ids.

Reply:
Done!